right_cam_info:
  image_width: 1920
  image_height: 1200
  camera_name: right
  camera_matrix:
    rows: 3
    cols: 3
    data: [1141.8338623046875, 0.0, 989.215576171875, 
              0.0, 1142.22412109375, 609.081787109375, 
              0.0, 0.0, 1.0]
  distortion_model: rational_polynomial
  distortion_coefficients:
    rows: 1
    cols: 8
    data: [3.5546460151672363, 9.322868347167969, 0.00039843941340222955, -0.00019973880262114108, 4.182700157165527, 3.5632097721099854, 9.054994583129883, 4.609309673309326]
  rectification_matrix:
    rows: 3
    cols: 3
    data: [ 0.9999575018882751, -0.005855862982571125, -0.007119601126760244, 
            0.005811530631035566, 0.9999637007713318, -0.006231606472283602, 
            0.0071558342315256596, 0.006189965642988682, 0.9999552369117737]
  projection_matrix:
    rows: 3
    cols: 4
    data: [1141.8338623046875, 0.0, 989.215576171875, 0.0, 
              0.0, 1142.22412109375, 609.081787109375, 0.0, 
              0.0, 0.0, 1.0, 0.0]